internal/acceptance/grant_test.go

```diff
@@ -104,3 +98,30 @@ func TestUcAccGrant(t *testing.T) {
 			Template: strings.ReplaceAll(grantTemplate, "%s", "{env.TEST_DATA_SCI_GROUP}"),
 		})
 }
+
+func grantTemplateForNameChange(suffix string) string {
+	return fmt.Sprintf(`
+	resource "databricks_storage_credential" "external" {
+		name = "cred-{var.STICKY_RANDOM}%s"
+		aws_iam_role {
+			role_arn = "{env.TEST_METASTORE_DATA_ACCESS_ARN}"
+		}
+		comment = "Managed by TF"
+	}
+	
+	resource "databricks_grant" "cred" {
+		storage_credential = databricks_storage_credential.external.id
+		principal  = "{env.TEST_DATA_ENG_GROUP}"
+		privileges = ["ALL_PRIVILEGES"]
+	}
+	`, suffix)
+}
+
+func TestUcAccGrantForIdChange(t *testing.T) {
```

Comment:
I tried running this test in debug mode, and it didn't seem to hit the CustomizeDiff codepath.